prices/price.py

```diff
@@ -14,6 +14,10 @@ def __init__(self, net, gross):
         if not isinstance(net, Amount) or not isinstance(gross, Amount):
             raise TypeError('Price requires two amounts, got %r, %r' % (
                 net, gross))
+        if net.currency != gross.currency:
+            raise ValueError(
+                'Net and gross requires same currency, got %r, %r' % (
```

Comment:
`amounts given in different currencies: %r and %r`

`ValueError: amounts given in different currencies: 'USD' and 'EUR'`

Reply:
Done